src/QUESTIONS.md

```diff
@@ -101,6 +134,80 @@ Of critical importance, you should upgrade your Morello board firmware before
 installing CheriBSD.
 
 
+### How can I upgrade CheriBSD?
+
+Currently, there are no binary upgrades for CheriBSD.
+
+The only way to upgrade a CheriBSD host is to build and install CheriBSD from
```

Comment:
I really do not want to be encouraging TAP participants to do this. The only official supported way to upgrade is to wipe and reinstall.

Reply:
I won't try to resolve this here, as the affected text was part of an earlier commit, [ca1bf1d](https://github.com/CTSRD-CHERI/cheri-faq/commit/ca1bf1dcd54af840fe74dfd106b6e09bc3dcc5f9), that I was looking to reposition. I've mentioned this issue here: #6.